src/main/io/osd.c

```diff
@@ -850,36 +896,43 @@ static void osdRefresh(timeUs_t currentTimeUs)
     }
 
```

Comment:
Please look in to slack, I've just send you diff with some fixes for statistics etc.